etc/global/cx/Surface.nl

```diff
@@ -1,5 +1,49 @@
 &CXControlNL
-cxfields_remove=23, 25, 31, 90, 1235, 3208, 3247, 3255, 3464, 3465, 8223, 8225, 9203, 9204, 9217, 9219, 16222, 20000, 20001, 20002
-cxfields_add=266, 267, 268
-
+! All variables below are found in OPS cx outputs for surface processing. Not all are
+! necessary for JOPA.
```

Comment:
Has this actually been checked in this case? If not I think the comment should be amended. The list is useful, I just don't want us to imply we've confirmed these are the fields that are output by OPS when we haven't actually and cause confusion down the line!

Reply:
Good point - taking a look...

Reply:
I have updated the table in be4a150. There were a few changes to make in the surface variables, and nothing for the upper-air ones.